src/quat.js

```diff
@@ -447,36 +447,50 @@ export function fromMat3(out, m) {
 }
 
 /**
- * Creates a quaternion from the given euler angle x, y, z.
+ * Creates a quaternion from the given euler angle z, y, x.
```

Comment:
In my opinion the docs should be explicit about the order that rotations are applied in. I think simply listing them here just looks like it's listing the argument order. It's also important to specify whether the convention is using [intrinsic or extrinsic rotations](https://en.wikipedia.org/wiki/Euler_angles#Chained_rotations_equivalence) regarding the order listed.

From my brief testing (please someone else verify -- I'd hate to be the one that causes some misdocumentation 😁) it seems the angles are applied in ZYX intrinsic order which is the same as XYZ extrinsic order. Either specification works for me.

Reply:
> In my opinion the docs should be explicit about the order that rotations are applied in.

Done.

> [I]t seems the angles are applied in ZYX intrinsic order [are] the same as XYZ extrinsic order

[This article seems to agree](https://dreamanddead.github.io/2019/04/24/understanding-euler-angles.html) with a proof. The proof can also be trivially modified to show that XYZ intrinsic order is equivalent to ZYX extrinsic order as well if you just change the subscripts.